css/style.css

```diff
@@ -16,4 +27,441 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* header */
+
+ header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2rem 5rem;
+ }
+
+ .logo{
```

Comment:
I would make sure that you're leaving a single whitespace between the selector and the declaration block consistently, there's tooling that you can use in VS Code that will take care of stuff like this for you automatically, but it's good to know the styling conventions anyway: https://www.w3schools.com/html/html5_syntax.asp